var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -224,6 +252,12 @@ def configure_args(self):
         else:
             args.append('--disable-cuda')
 
+        if '+regcache' in self.spec:
+            args.append('--enable-registration-cache')
```

Comment:
Just to let you know of the feature, in case you were unaware of it. If you called the variant `registration-cache`, you could have done something like:
```python
args.extend(self.enable_or_disable('registration_cache'))
```
instead of these four lines.